Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.cs

```diff
@@ -123,7 +123,7 @@ protected virtual double CalculateItemWidth(double containerWidth)
                 _needContainerMarginForLayout = true;
             }
 
-            return (containerWidth / columns) - itemMargin.Left - itemMargin.Right;
+            return (containerWidth / columns) - itemMargin.Left - itemMargin.Right - 1;
```

Comment:
This is the same change that caused #1737

Reply:
Good call, will test that scenario and see if I can fix both.